modules/kueue-viewing-pending-workloads-clusterqueue.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/kueue/monitoring-pending-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-pending-workloads-clusterqueue_{context}"]
+= Viewing pending workloads in ClusterQueue
+
+[role=”_abstract”]
+A ClusterQueue is a cluster-scoped resource that governs a pool of resources, such as pods, CPU, memory, and hardware accelerators. Use ClusterQueues to define usage limits usage limits and order of consumption. You can also define the API Priority and Fairness (APF) rules across the multiple ClusterQueues in the cluster.
```

Comment:
```suggestion
A ClusterQueue is a cluster-scoped resource that governs a pool of resources, such as pods, CPU, memory, and hardware accelerators. Use ClusterQueues to define usage limits and order of consumption. You can also define the Fair Sharing rules across the multiple ClusterQueues in the cluster.
```

Reply:
Even better is to remove this part, which I don't think is necessary:
`Use ClusterQueues to define usage limits and order of consumption. You can also define the Fair Sharing rules across the multiple ClusterQueues in the cluster.`

Reply:
Removed.